src/preact-render-spy.js

```diff
@@ -5,70 +5,105 @@ const {isWhere} = require('./is-where');
 const {selToWhere} = require('./sel-to-where');
 
 const SPY_PRIVATE_KEY = 'SPY_PRIVATE_KEY';
```

Comment:
I feel like we should:

```js
export const config = {
  SPY_PRIVATE_KEY: 'SPY_PRIVATE_KEY',
};
```

just in case someone wants to:

```js
import {config} from 'preact-render-spy';
config.SPY_PRIVATE_KEY='@@somethingelse';